frontend/app/src/pages/main/events/components/event-columns.tsx

```diff
@@ -78,13 +78,21 @@ export const columns = ({
         return <div>{relativeDate(row.original.metadata.createdAt)}</div>;
       },
     },
+    // empty columns to get column filtering to work properly
     {
       accessorKey: 'workflows',
       header: () => <></>,
       cell: () => {
         return <div></div>;
       },
     },
+    {
+      accessorKey: 'status',
+      header: () => <></>,
+      cell: () => {
+        return <div></div>;
```

Comment:
empty div?

Reply:
we should consider using the cell output so we can make these tables dynamic at some point

Reply:
Yep, see comment above.